dependencies.yaml

```diff
@@ -126,39 +126,45 @@ dependencies:
       - output_types: conda
         packages:
           - &doxygen doxygen=1.9.1
-  cudatoolkit:
```

Comment:
The inclusion of `cudatoolkit` is now handled by a separate dependency list called `cuda`. It turns out that the dependency files where the `cudatoolkit` dependency list was being included should already get their required CUDA parts as dependencies of the built `rmm` packages (for test environments) or `cuda-nvcc` itself (for using `build.sh` with `conda/environments/all_cuda-120_arch-x86_64.yaml`, see also https://github.com/rapidsai/rmm/pull/1422/files#r1446707857).